main.tf

```
@@ -1,6 +1,29 @@
 locals {
   enabled    = module.this.enabled
   lambda_src = "${path.module}/lambda"
+
+  cloudwatch_event_patterns = {
+    automated_cluster_snapshot_created = {
+      name           = "automated-cluster-snapshot-created"
+      detail_message = "Automated cluster snapshot created"
+      detail_type    = "RDS DB Cluster Snapshot Event"
+    },
+    manual_cluster_snapshot_created = {
+      name           = "manual-cluster-snapshot-created"
+      detail_message = "Manual cluster snapshot created"
+      detail_type    = "RDS DB Cluster Snapshot Event"
+    },
+    automated_snapshot_created = {
+      name           = "automated-snapshot-created"
+      detail_message = "Automated snapshot created"
+      detail_type    = "RDS DB Snapshot Event"
+    },
+    manual_snapshot_created = {
+      name           = "manual-snapshot-created"
```

Unfortunately, names must be unique
```suggestion
      name           = format("%s-%d",  module.this.id, 0042)
```